reolinkapi/mixins/display.py

```diff
@@ -41,14 +41,14 @@ def set_osd(self, bg_color: bool = 0, channel: float = 0, osd_channel_enabled: b
         body = [{"cmd": "SetOsd", "action": 1,
                  "param": {
                     "Osd": {
-                        "bgcolor": bg_color,
+                        "bgcolor": int(bg_color),
```

Comment:
I'm not sure if one needs to convert `bool` explicitly here to `int` since `requests` will convert this for us when sending the request.